apps/state/test/state/prediction_test.exs

```diff
@@ -86,6 +86,18 @@ defmodule State.PredictionTest do
 
       assert [%{direction_id: 1}] = pre_insert_hook(prediction)
     end
+
+    test "sets trip_match? to true if there's a trip in GTFS" do
```

Comment:
Might also be worth testing that `trip_match?` doesn't get set if there's no trip?

Reply:
9463e09